src/patternfly/components/Table/table.scss

```diff
@@ -390,10 +391,20 @@
 .pf-c-table__sort {
   // override pf-m-plain color
   .pf-c-button {
+    // stylelint-disable
     &.pf-m-plain {
       color: var(--pf-c-table__sort--c-button--Color);
     }
+
+    &.pf-m-sort-wrap {
+      display: flex;
+
+      .pf-c-table__sort--c-button-text {
+        white-space: normal;
+      }
+    }
   }
```

Comment:
I think you can achieve the same result without having to create a new selector. Does this work for you?

```
// Table sort
// ==================================================================
.pf-c-table__sort {
  // override pf-m-plain color
  .pf-c-button {
    // stylelint-disable
    &.pf-m-plain {
      color: var(--pf-c-table__sort--c-button--Color);
    }
  }

  &.pf-m-wrap .pf-c-button {
    white-space: normal;
    display: flex;
  }
  // stylelint-enable

  &.pf-m-selected {
    // set button and indicator color
    .pf-c-button,
    .pf-c-table__sort-indicator {
      color: var(--pf-c-table__sort--sorted--Color);
    }
  }

  &:not(.pf-m-selected) {
    &:hover {
      .pf-c-table__sort-indicator {
        color: var(--pf-c-table__sort-indicator--hover--Color);
      }
    }
  }
}
```